rllib/examples/multi_agent_cartpole.py

```diff
@@ -75,12 +75,21 @@
 
     # Each policy can have a different configuration (including custom model).
     def gen_policy(i):
-        config = PPOConfig.overrides(
-            model={
-                "custom_model": ["model1", "model2"][i % 2],
-            },
-            gamma=random.choice([0.95, 0.99]),
-        )
+
+        if bool(os.environ.get("RLLIB_ENABLE_RL_MODULE", False)):
```

Comment:
Can we instead use a command line arg here, like we do for all other settings?
I know we have the gpu env setting, but te reason for that is to be able to set this globally so we can more gracefully handle this case in our `buildkite.yml` file. But for this case here, I don't see that need. Thx!

Reply:
I have to disagree due to the reason above. I went through both solutions and found this one to be more scalable for our transition.